system-contracts/contracts/EvmInterpreterFunctions.template.yul

```diff
@@ -470,10 +465,10 @@ function checkMemOverflow(location) {
 }
 
 // Note, that this function can overflow. It's up to the caller to ensure that it does not.
-function memCost(memSizeWords) -> gasCost {
+/*function memCost(memSizeWords) -> gasCost {
```

Comment:
Let's delete this function if we're not using it

Reply:
Done!